incubator/couchdb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: couchdb
-version: 0.1.7
-appVersion: 2.1.1
+version: 0.1.8
```

Comment:
@kocolosk ping

Reply:
I am OK with that - will update the PR now.

Reply:
I just bumped minor rather than major as it seems premature to declare 1.0.0 on a chart in incubator/. I think that's what you had in mind anyway, right @davidkarlsen ?